pkg/instancetype/instancetype_test.go

```diff
@@ -304,7 +304,7 @@ var _ = Describe("Instancetype and Preferences", func() {
 				vm.Spec.Template.Spec.Domain.CPU = &v1.CPU{
 					Cores: 1,
 				}
-				Expect(instancetypeMethods.StoreControllerRevisions(vm)).To(MatchError(Equal(fmt.Sprintf(instancetypeErrors.VMFieldsConflictsErrorFmt, "spec.template.spec.domain.cpu.cores"))))
+				Expect(instancetypeMethods.StoreControllerRevisions(vm)).To(MatchError(Equal(conflict.New("spec", "template", "spec", "domain", "cpu", "cores").Message())))
```

Comment:
```suggestion
				Expect(instancetypeMethods.StoreControllerRevisions(vm)).To(MatchError(conflict.New("spec", "template", "spec", "domain", "cpu", "cores").Message()))
```

Equal should not be required.

Reply:
Same below...